packages/flutter/android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="technology.breez.spark">
+        <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
What is this used for?

Reply:
I reverted this, was just for trying to resolve the Android issue [77a34a8](https://github.com/breez/spark-sdk/pull/204/commits/77a34a8947b59d7a9a1210fe922875daaee2b1ab)